clair-in-ci/Dockerfile

```diff
@@ -1,3 +1,3 @@
 FROM quay.io/crozzy/clair-action:v0
-RUN microdnf install wget zstd -y && wget -q https://clair-sqlite-db.s3.amazonaws.com/matcher.zst && zstd -o /tmp/matcher.db -d matcher.zst && rm -f matcher.zst
-RUN microdnf clean all
```

Comment:
I got a feeling I needed to separate install with clean because it was causing troubles, but I hope I am wrong with this one.

Reply:
it shouldn't it separated by `&&`

Reply:
Builds works and DB is in the final image
```
[root@b766ace231bc /]# ls -l /tmp/matcher.db 
-rw-r--r--. 1 root root 13520171008 Jan 20 05:29 /tmp/matcher.db
```

Reply:
Also this would generate even smaller image :D but IMO this is good enough
```
FROM ubi/ubi8-minimal as builder
RUN microdnf install wget zstd -y && wget -q https://clair-sqlite-db.s3.amazonaws.com/matcher.zst && zstd -o /tmp/matcher.db -d matcher.zst && rm -f matcher.zst

FROM quay.io/crozzy/clair-action:v0
COPY --from=builder /tmp/matcher.db /tmp/matcher.db
```